### PR TITLE
[BAHIR-291] Bump flink to 1.14.0

### PR DESCRIPTION
**Pull request by RocMarshal**

### CODE REVIEW

**Comment by eskabetxe** [comment]
Thanks @RocMarshal for the contribution, could you check the failing tests?

**Comment by RocMarshal** [comment]
> Thanks @RocMarshal for the contribution, could you check the failing tests?

@eskabetxe Glad to get your attention . 😊   I'll check the failing tests ASAP.

**Comment by RocMarshal** [comment]
@eskabetxe I made some change. Would you mind checking it ? Thx.

File: flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/common/RedisSinkZIncrByTest.java [hunk 1]
```diff
@@ -1,121 +0,0 @@
-/*
```
Comment (eskabetxe) [inline flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/common/RedisSinkZIncrByTest.java:1]: why we delete this test?

File: flink-connector-activemq/pom.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@ under the License.
 
     <!-- Allow users to pass custom connector versions -->
     <properties>
-        <activemq.version>5.15.10</activemq.version>
+        <activemq.version>5.16.2</activemq.version>
```
Comment (eskabetxe) [inline flink-connector-activemq/pom.xml:38]: is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)

File: flink-connector-flume/pom.xml [hunk 3]
```diff
@@ -36,7 +36,7 @@ under the License.
 	<!-- Allow users to pass custom connector versions -->
 	<properties>
 		<flume-ng.version>1.9.0</flume-ng.version>
-		<testcontainers.version>1.15.3</testcontainers.version>
+		<testcontainers.version>1.16.2</testcontainers.version>
```
Comment (eskabetxe) [inline flink-connector-flume/pom.xml:39]: is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)

File: flink-connector-activemq/pom.xml [hunk 4]
```diff
@@ -101,14 +101,14 @@ under the License.
         <dependency>
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-junit4</artifactId>
-            <version>1.5.5</version>
+            <version>2.0.9</version>
```
Comment (eskabetxe) [inline flink-connector-activemq/pom.xml:104]: is this mandatory for flink update?
if not will be better in other commit, only for powermock update (and related)

File: flink-connector-influxdb2/pom.xml [hunk 5]
```diff
@@ -39,7 +39,7 @@ under the License.
     <influxdbClient.version>2.0.0</influxdbClient.version>
     <druid.version>0.13.0-incubating</druid.version>
     <!--  Test Properties  -->
-    <testcontainers.version>1.15.2</testcontainers.version>
+    <testcontainers.version>1.16.2</testcontainers.version>
```
Comment (eskabetxe) [inline flink-connector-influxdb2/pom.xml:42]: is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)

File: flink-connector-redis/pom.xml [hunk 6]
```diff
@@ -34,7 +34,7 @@ under the License.
     <packaging>jar</packaging>
 
     <properties>
-        <jedis.version>2.9.0</jedis.version>
+        <jedis.version>3.7.1</jedis.version>
```
Comment (eskabetxe) [inline flink-connector-redis/pom.xml:37]: is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)

File: flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisDescriptorTest.java [hunk 7]
```diff
@@ -51,23 +46,26 @@ public void testRedisDescriptor() throws Exception {
...
         EnvironmentSettings settings = EnvironmentSettings
                 .newInstance()
-                .useOldPlanner()
                 .inStreamingMode()
                 .build();
         StreamTableEnvironment tableEnvironment = StreamTableEnvironment.create(env, settings);
         tableEnvironment.registerDataStream("t1", source, "k, v");
 
-        Redis redis = new Redis()
+        /*Redis redis = new Redis()
```
Comment (eskabetxe) [inline flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisDescriptorTest.java:54]: this comment could be removed no?

**Review by eskabetxe — CHANGES_REQUESTED** [review]
@RocMarshal overall looks good, all bump version if not related with flink update, will be better in specific commit for each one

**Comment by eskabetxe** [comment]
@RocMarshal thanks for contribution.. I fix the failing test and merged the PR

**Pull request merged**